app/models/additional_cost.rb

```diff
@@ -1,3 +1,11 @@
 class AdditionalCost < ApplicationRecord
   belongs_to :prior_authority_application
+
+  def total_cost
```

Comment:
I think we're better off reusing our existing logic here given my comment from my previous review. How about:

```ruby
PriorAuthority::Steps::AdditionalCosts::DetailForm.build(self, application: prior_authority_application).total_cost
```